### ISSUE TITLE
Collector version changes

### ISSUE THREAD

**Issue by anatula** [pinojs/pino-opentelemetry-transport#191]
Hello, the usage example in the [README.md](https://github.com/pinojs/pino-opentelemetry-transport/blob/main/README.md#usage) works fine using `v0.103.1` of the collector (as specified in the [docker-compose.yaml](https://github.com/pinojs/pino-opentelemetry-transport/blob/main/docker-compose.yaml)). If you pull the collector using `latest` tag it downloads `v0.112.0` and [otel-collector-config.yaml](https://github.com/pinojs/pino-opentelemetry-transport/blob/main/otel-collector-config.yaml) need the following changes:

- Since [v0.104.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.104.0), otlpreceiver now uses localhost by default instead of 0.0.0.0 -> for **testing** purposes explicitly set the endpoint to 0.0.0.0

- Since [v0.111.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.111.0), the core release removes the logging exporter -> use the debug exporter

So, the `otel-collector-config.yaml` now becomes:

```
receivers:
  otlp:
    protocols:
      grpc:
        endpoint: 0.0.0.0:4317
      http:
        endpoint: 0.0.0.0:4318

exporters:
  file:
    path: ./etc/test-logs/otlp-logs.log
    flush_interval: 1

  debug:
    verbosity: detailed
  
processors:
  batch:

service:
  pipelines:
    logs:
      receivers: [otlp]
      processors: []
      exporters: [debug, file]
```

Hope this helps someone, thank you!

**Comment by Vunovati** [comment]
Thank you for the tip. I was aware that there had been some changes in the collector but had not had time to act on them yet. I'll try to incorporate them when I get some time. Feel free to create a PR yourself.

**Comment by Vunovati** [comment]
Closed with contribution from @anatula #193
